.github/workflows/ci.yaml

```diff
@@ -10,38 +10,25 @@ on:
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.8]
+        python-version: [3.9]
 
     steps:
     - uses: actions/checkout@v2
     - name: Set up Python ${{ matrix.python-version }}
       uses: actions/setup-python@v2
       with:
         python-version: ${{ matrix.python-version }}
-
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install flake8 black
-        pip install -r requirements.txt
-        pip install -r requirements-dev.txt
-    - name: Lint with flake8
+        pip install pre-commit pytest
+    - name: Run pre-commit
       run: |
-        # stop the build if there are Python syntax errors or undefined names
-        flake8 notify --count --select=E9,F63,F7,F82 --show-source --statistics
-        # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
-        flake8 notify --count --max-complexity=10 --max-line-length=120 --statistics --per-file-ignores="__init__.py:F401"
+        pre-commit run --all-files
     - name: Test with pytest
-      env:
-        AZURE_CLIENT_ID: ${{ secrets.AZURE_CLIENT_ID }}
-        AZURE_CLIENT_SECRET: ${{ secrets.AZURE_CLIENT_SECRET }}
-        AZURE_TENANT_ID: ${{ secrets.AZURE_TENANT_ID }}
       run: |
+        pip install -r requirements.txt
```

Comment:
@erfannariman Pytest faalt omdat keyvault niet geinstalleerd is, die zit in requirements-dev.txt, die werd eerst wel geinstalleerd. Is dat met een reden eruit gehaald?

Reply:
This is true, and I have the change ready, but didnt want to push again since the mails will be sent. But will do it now anyways

Reply:
CI is green now.